.github/workflows/awscurl-build.yml

```diff
@@ -0,0 +1,47 @@
+name: awscurl build
+
+on:
+  workflow_dispatch:
+
+jobs:
+  create-runners:
+    runs-on: [ self-hosted, scheduler ]
+    steps:
+      - name: Create new Graviton instance
+        id: create_aarch64
```

Comment:
Why need aarch64?

We should just update djl-serving publish pipeline in djl repo